web/satellite/vuetify-poc/src/styles/styles.scss

```diff
@@ -459,3 +459,17 @@ table {
 .no-position {
   position: unset !important;
 }
+
+// Login and Signup Background
+.signup-background::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-image: url('../assets/bg.svg');
+  background-size: cover;
```

Comment:
NIT - can you try a smaller background size? `background-size: auto 75%;` for example.
Let me know what you think.